### PR TITLE
`sage.ext.fast_callable`: Docstring cosmetics

### PR DESCRIPTION
**Pull request by mkoeppe**
- Cherry-picked from #35095 

### :memo: Checklist

- [x] The title is concise and informative.
- [ ] The description explains in detail what this PR is about.
- [x] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

File: src/sage/ext/fast_callable.pyx [hunk 1]
```diff
@@ -2331,17 +2344,18 @@ cdef class InstructionStream:
...
-    See the class docstring for CompilerInstrSpec for more information.
+    The interpreter metadata for a :func:`fast_callable` interpreter.
+
+    Currently consists of a dictionary mapping instruction names to
+    (:class:`CompilerInstrSpec`, opcode) pairs, a list mapping opcodes to
+    (instruction name, :class:`CompilerInstrSpec`) pairs, and a range of exponents
+    for which the ``'ipow'`` instruction can be used.  This range can be
+    ``False`` (if the ``'ipow'`` instruction should never be used), a pair of
+    two integers ``(a, b)``, if ``'ipow'`` should be used for a<=n<=b, or ``True``,
+    if ``'ipow'`` should always be used.  When ``'ipow'`` cannot be used, then
```
Comment (kwankyu) [inline src/sage/ext/fast_callable.pyx]: both `(a,b)` and `a<=n<=b` math or code?

File: src/sage/ext/fast_callable.pyx [hunk 2]
```diff
@@ -1713,6 +1722,7 @@ class IntegerPowerFunction():
         """
         return x**self.exponent
 
+
```
Comment (kwankyu) [inline src/sage/ext/fast_callable.pyx:1724]: Treat `get_builtin_functions()` from line 1729?

**Review by gmou3 — APPROVED** [review]

File: src/sage/ext/fast_callable.pyx (hunk 2 above)
Comment (mkoeppe) [reply src/sage/ext/fast_callable.pyx:1724]: Done in c3093443772

**Review by kwankyu — APPROVED** [review]
LGTM.

**Comment by mkoeppe** [comment]
Thanks both!

**Pull request merged**